### PR TITLE
RDoc-2562 Cloud ->Microsoft Azure Marketplace - prepare a documentation about the integration between RavenDB Cloud and Azure Marketplace

### PR DESCRIPTION
**Pull request by PFYasu**
Ticket: https://issues.hibernatingrhinos.com/issue/RDoc-2562/Prepare-a-documentation-about-the-integration-between-RavenDB-Cloud-and-Azure-Marketplace

### CODE REVIEW

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-microsoft-azure-marketplace.markdown [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+{NOTE/}
+
+---
+
+{PANEL: Overview}
+
+If you want to pay for *RavenDB Cloud* services using a credit card associated with your *Microsoft Azure* account, without re-entering the card in *RavenDB Cloud*, you can subscribe to [RavenDB Cloud offer](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/hibernatingrhinos.ravendb-cloud-payg?tab=Overview) on the *Microsoft Azure Marketplace*.
+
+The *Microsoft Azure Cloud Accounts* created by this integration have a few differences compared to the regular *Cloud Accounts*:
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-microsoft-azure-marketplace.markdown:21]:
```suggestion
*Microsoft Azure Cloud Account* created this way has a few differences in comparison to the regular *Cloud Account*:
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-microsoft-azure-marketplace.markdown [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+- Monthly invoices are issued by *Microsoft Azure*, not by our standard billing system.
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-microsoft-azure-marketplace.markdown:23]:
```suggestion
- Monthly invoices are issued by *Microsoft Azure*, not by RavenDB Cloud
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-microsoft-azure-marketplace.markdown [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+- *Microsoft Azure Cloud Accounts* are permitted to provision only *Azure* products. Regular Cloud Accounts can also provision *AWS* and *GCP* products.
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-microsoft-azure-marketplace.markdown:24]:
```suggestion
- *Microsoft Azure Cloud Account* offering is limited to *Azure* products.
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-microsoft-azure-marketplace.markdown [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+1.  **Subscription**  
+    Select the *Microsoft Azure subscription* from which your payment information will be used for this offer.
+
+2.  **Resource group**  
+    Select the *Microsoft Azure resource group* where the subscription will be located.
+
+3.  **Name**  
+    Provide a name for the subscription. 
+
+4.  **Recurring billing**
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-microsoft-azure-marketplace.markdown:59]: Let's put an asterisk on it for attention? Thoughts?
```suggestion
4.  **Recurring billing**  *
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-microsoft-azure-marketplace.markdown [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+
+3.  **Name**  
+    Provide a name for the subscription. 
+
+4.  **Recurring billing**  
+    When recurring billing is on, your subscription will renew at the end of the billing term.
+
+{DANGER: }
+If **recurring billing** option is **off**, your subscription **won't be renewed at the end of the billing term**.
+This means your subscription, account and related Cloud products **will be permanently lost** at the end of the billing term.
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-microsoft-azure-marketplace.markdown:64]:
```suggestion
This means your subscription, account along with its Cloud products and data they held, **will be permanently lost** at the end of the billing term.
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-microsoft-azure-marketplace.markdown [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+!["Figure 6 - RavenDB Cloud Azure Marketplace Landing Page"](images\cloud-ravendb-azure-marketplace-landing-page.png "Figure 6 - RavenDB Cloud Azure Marketplace Landing Page")
+
+1.  **Domain Name**  
+    Provide a Domain Name for the account.
+
+2.  **Account Type**  
+    Provide an Account Type for the account.
+
+3.  **T&C and privacy policy**  
+    Accept Terms & Conditions and our privacy policy.
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-microsoft-azure-marketplace.markdown:105]:
```suggestion
    Accept Terms & Conditions and the privacy policy.
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-microsoft-azure-marketplace.markdown [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+
+After providing the information, please click **Sign up**.
+
+---
+
+Once you have done this, your account is ready to use. To sign in to the account, click **Continue to sign in**.
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-microsoft-azure-marketplace.markdown:111]:
```suggestion
Your account is now ready to use. To sign in to the account, click **Continue to sign in**.
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-microsoft-azure-marketplace.markdown (hunk 4 above)
Comment (PFYasu) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-microsoft-azure-marketplace.markdown:59]: Ok, added.

**Review by gregolsky — APPROVED** [review]

**Pull request merged**